grafast/website/grafserv/servers/h3.md

```diff
@@ -26,31 +31,9 @@ serv.addTo(app).catch((e) => {
   process.exit(1);
 });
 
+// If you need websocket subscriptions (need h3@^1.13.0);
+app.use("/path_to_ws_handler", defineWebSocketHandler(serv.makeWsHandler()));
```

Comment:
I do this because there is no option in grafserv for pass the ws endpoint (which isn't necessarily the same, but it can).
Yes, we can delegate this in `addTo` function.
I'll make theses changes.

Reply:
Hmmm; if that's a useful feature then lets add it here:

https://github.com/graphile/crystal/blob/44ca3dadfa05a86da50875326009fa45662b5aa1/grafast/grafserv/src/index.ts#L109-L112

Something like:

```ts
      /** The path at which the GraphQL websocket would be made available; defaults to `graphqlPath` */
      websocketsPath?: string;
```

Then we should adjust the relevant places to support this option.

Alternatively for now just require that it matches `graphqlPath` and we can change it later.

Reply:
(I think the latter is what you have done?)

Reply:
As ruru does not yet support a custom ws endpoint: [/grafast/grafserv/src/middleware/graphiql.ts#L37-L39](https://github.com/graphile/crystal/blob/10ed7e83767e0552c3132aeb73d27e66d7d5fc7b/grafast/grafserv/src/middleware/graphiql.ts#L37-L39), it's relevant for now to require it to match graphqlPath and we can change it later 👍 

> (I think the latter is what you have done?)

Yes

Reply:
We'll save the other option for a later PR then.